unittest/src/fake/leds/tests.rs

```diff
@@ -0,0 +1,45 @@
+use crate::fake;
+use fake::leds::*;
+
+// Tests the command implementation.
+#[test]
+fn command() {
+    use fake::SyscallDriver;
+    let leds = Leds::<10>::new();
+    let value = leds.command(DRIVER_CHECK, 1, 2);
+    assert!(value.is_success_u32());
```

Comment:
The assert for `is_success_u32()` is redundant, because `assert_eq!(value.get_success_u32(), Some(10));` will indicate the correct error if a different type is returned.